data-structures-stacks/insert-under-top.js

```diff
@@ -0,0 +1,14 @@
+/* exported insertUnderTop */
+
+function insertUnderTop(stack, value) {
+  const newStack = stack;
+
+  if (newStack.peek() === undefined) {
+    return undefined;
+  } else {
```

Comment:
Since there is a `return` in the above `if` statement you don't need an `else` here